odl/solvers/functional/default_functionals.py

```diff
@@ -2193,6 +2198,169 @@ def __repr__(self):
                                          self.__norm.pwisenorm.exponent)
 
 
+class IndicatorSimplex(Functional):
+
+    """The indicator functional of a simplex in any dimension.
```

Comment:
Can you use the `r"""` docstring style here please? Saves a lot of double backslashes.